src/video/wayland/SDL_waylandwindow.c

```diff
@@ -958,6 +1101,13 @@ Wayland_SetWindowMinimumSize(_THIS, SDL_Window * window)
     SDL_WindowData *wind = window->driverdata;
 
     if (window->flags & SDL_WINDOW_RESIZABLE) {
+#ifdef HAVE_LIBDECOR_H
+        if (data->shell.libdecor) {
+        libdecor_frame_set_min_content_size(wind->shell_surface.libdecor.frame,
+                                            window->min_w,
+                                            window->min_h);
```

Comment:
Latest pass. Why do I keep missing these...

```suggestion
            libdecor_frame_set_min_content_size(wind->shell_surface.libdecor.frame,
                                                window->min_w,
                                                window->min_h);
```